wandb/run_manager.py

```diff
@@ -431,6 +431,7 @@ def __init__(self, api, run, project=None, tags=[], cloud=True, output=True, por
         self._system_stats = stats.SystemStats(run, api)
         self._meta = meta.Meta(api, self._run.dir)
         self._meta.data["jobType"] = self._run.job_type
+        self._meta.data["mode"] = self._run.mode
```

Comment:
Storing mode so we can later selectively only sync dryrun runs when a user isn't logged in.